src/styles/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  height: 408px;
+  width: 200px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  border: 1px solid $background-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__bottom {
+    width: 166px;
```

Comment:
```suggestion

```
Consider setting left and right paddings for the card block to limit the width of child blocks